compiam/rhythm/transcription/mnemonic_transcription/__init__.py

```diff
@@ -0,0 +1,314 @@
+import csv
+import os
+import pickle
+
+import hmmlearn.hmm as hmm
+import librosa
+import numpy as np
+from sklearn.exceptions import NotFittedError
+import tqdm 
+
+from compiam.exceptions import ModelNotTrainedError
+from compiam.rhythm.meter.akshara_pulse_tracker import AksharaPulseTracker
+from compiam.utils import get_logger
+
+logger = get_logger(__name__)
+
+class MnemonicTranscription:
+    """
+    bōl or solkattu transcription from audio. Based on model presented in [1]
+    
+    [1] Gupta, S., Srinivasamurthy, A., Kumar, M., Murthy, H., & Serra, X. 
+    (2015, October). Discovery of Syllabic Percussion Patterns in Tabla 
+    Solo Recordings. In Proceedings of the 16th International Society 
+    for Music Information Retrieval Conference (ISMIR 2015) (pp. 385–391). 
+    Malaga, Spain.
```

Comment:
Maybe this is not needed given that the citation is already in the README and we do not add citations to any other docstring, but I guess it's not a big problem to have it here. Maybe just a single time, since it appears twice: here and then in the `__init__` as well.